src/components/App.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import UserInfo from "./userComponent/userComponent";
```

Comment:
change the name of this folder from userComponent to just user so that your path changes from `"./userComponent/userComponent"` to `"./user/userComponent"`